cuML/src/knn/knn_c.h

```diff
@@ -0,0 +1,31 @@
+
```

Comment:
Missing header file. Also this is not a C file, why not change the name to something more appropriate? Can be in one of the follow up PRs to speed this (the file name it is).